tests/test_ha.py

```diff
@@ -1027,7 +1027,6 @@ def test_update_cluster_history(self):
             self.assertEqual(self.ha.run_cycle(), 'no action.  i am the leader with the lock')
 
     @patch('sys.exit', return_value=1)
-    @patch('requests.get', requests_get)
```

Comment:
is it an unrelated change that simply removes a no longer needed mock ?

Reply:
Well, actually it is related. Because it tests the [code](https://github.com/zalando/patroni/pull/1217/files#diff-57c53e988b775ee2902488fb33c1f433R1287-R1294) I was changing.